src/detect-http-server-body.c

```diff
@@ -124,6 +124,9 @@ static int DetectHttpServerBodySetupSticky(DetectEngineCtx *de_ctx, Signature *s
         return -1;
     if (DetectSignatureSetAppProto(s, ALPROTO_HTTP) < 0)
         return -1;
+    // file data is on both directions, but we only take the one to client here
+    s->flags |= SIG_FLAG_TOCLIENT;
+    s->flags &= ~SIG_FLAG_TOSERVER;
```

Comment:
I tried to do as `flow:to_client`

I did not want to error on `s->flags & SIG_FLAG_TOSERVER` as we can have `s->flags = SIG_FLAG_TOSERVER | SIG_FLAG_TOCLIENT`
But not 100% sure...

Reply:
I think it makes sense.

Could you add a SV test that checks the engine analysis as well? Similar to https://github.com/OISF/suricata-verify/tree/master/tests/rules